website/docs/reference/warehouse-setups/databricks-setup.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Databricks setup"
+id: "databricks-setup"
+meta:
+  maintained_by: Databricks
+  authors: 'some dbt loving Bricksters'
+  github_repo: 'databricks/dbt-databricks'
+  pypi_package: 'dbt-databricks'
+  min_core_version: 'v0.18.0'
+  cloud_support: Coming Soon
+  min_supported_version: 'n/a'
+  slack_channel_name: '#db-databricks-and-spark'
+  slack_channel_link: 'https://getdbt.slack.com/archives/CNGCW8HKL'
+  platform_name: 'Databricks'
+  config_page: 'spark-configs'
+---
+
+<h2> Overview of {frontMatter.meta.pypi_package} </h2>
+
+<ul>
+    <li><strong>Maintained by</strong>: {frontMatter.meta.maintained_by}</li>
+    <li><strong>Authors</strong>: {frontMatter.meta.authors}</li>
+    <li><strong>GitHub repo</strong>: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a><a href={`https://github.com/${frontMatter.meta.github_repo}`}><img src={`https://img.shields.io/github/stars/${frontMatter.meta.github_repo}?style=for-the-badge`}/></a></li>
+    <li><strong>PyPI package</strong>: <code>{frontMatter.meta.pypi_package}</code> <a href={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}`}><img src={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}.svg`}/></a></li>
+    <li><strong>Slack channel</strong>: <a href={frontMatter.meta.slack_channel_link}>{frontMatter.meta.slack_channel_name}</a></li>
+    <li><strong>Supported dbt Core version</strong>: {frontMatter.meta.min_core_version} and newer</li>
+    <li><strong>dbt Cloud support</strong>: {frontMatter.meta.cloud_support}</li>
+    <li><strong>Minimum data platform version</strong>: {frontMatter.meta.min_supported_version}</li>
+    </ul>
+
+## Installation and Distribution
+
+
+<h2> Installing {frontMatter.meta.pypi_package} </h2>
+
+The easiest way to install the adapter is to use pip:
+
+<code>pip install {frontMatter.meta.pypi_package}</code>
+
+<p>You don't need to install dbt separately. Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> and any other dependencies.</p>
+
+<h2> Configuring {frontMatter.meta.pypi_package} </h2>
+
+<p>For {frontMatter.meta.platform_name}-specifc configuration please refer to <a href={frontMatter.meta.config_page}>{frontMatter.meta.platform_name} Configuration</a> </p>
+
+<p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
+
+
+### Set up a Databricks Target
+
+dbt-databricks can connect to Databricks all-purpose clusters as well as SQL endpoints.
+The latter provides an opinionated way of running SQL workloads with optimal performance and
```

Comment:
```suggestion
The latter provides an opinionated way of running SQL workloads with optimal performance and price; the former provides all the flexibility of Spark.
```

Reply:
great question -- the idea is that it's pre-configured to work better out-of-the-box compared to a traditional Spark cluster